migrations/20200414062630-augment-resolved-connected.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn(
+      'Availabilities',
+      'resolved',
+      {
+        type: Sequelize.BOOLEAN
+      }
+    );
+    await queryInterface.addColumn(
+      'Availabilities',
+      'connected',
```

Comment:
I'm assuming connected implies that the request (application) has been resolved by our volunteers

Reply:
Not quite. The volunteers required 2 kinds of status

- connected: whether the application has been matched with some other application. Like a requirement being matched with a manufacturer
- resolved: the application was completely resolved, i.e. PPE was actually transferred from one party to another. Since we have no way to automatically track this, volunteers have to manually check.